plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -883,6 +898,7 @@ private val PageNode.isNavigable: Boolean
     get() = this !is RendererSpecificPage || strategy != RenderingStrategy.DoNothing
 
 private fun PropertyContainer<ContentNode>.extraHtmlAttributes() = allOfType<SimpleAttr>()
+private fun PropertyContainer<ContentNode>.extraToggleableContentType() = allOfType<ToggleableContentTypeExtra>().lastOrNull()
```

Comment:
The rule of style applying.

Reply:
But it's not a style, it's an attribute. With styles it makes sense because CSS rules are sequential.

If someone else implements another format, how do they know that they only need to take the last element from this list? What happens if you apply all of them?

Reply:
It is a general question: How do they know that they only need to take the (first, last, single, none and so on) element from this list?
You can not know it without having a look at the implementation.

> What happens if you apply all of them?

It depends on implementation.  
Anyway, this is ambiguous. Actually, we have only one.

Reply:
I'm sorry, but I still don't understand. I'm looking at the implementation and nothing tells me that I should only be applying the last `ToggleableContentTypeExtra` if there are multiple. What happens if I change `.lastOrNull()` to `.singleOrNull()`?

Reply:
It has been written, now only one `ToggleableContentTypeExtra` is possible. 

> hat happens if I change .lastOrNull() to .singleOrNull()? 

Now nothing

> How would the API consumer know about this if they start implementing tabs in another format?

it is a general problem of the design of `extra`.
 `style`  has this kind of problem too.

Reply:
If `lastOrNull()` exists for no reason, let's remove it? I'm pretty sure you can do this:

```kotlin
private fun PropertyContainer<ContentNode>.extraToggleableContentType() = this[ToggleableContentTypeExtra]
```